exercises/concept/wizards-and-warriors/.docs/instructions.md

```diff
@@ -45,6 +45,8 @@ Implement the `Wizard.PrepareSpell()` method to allow a Wizard to prepare a spel
 ```csharp
 var wizard = new Wizard();
 wizard.PrepareSpell();
+wizard.Vulnerable();
```

Comment:
I don't think that the vulnerability of Wizards should be a concern at that point.
Since the next task of the exercise is setting the wizard to be not vulnerable after preparing a spell, at this point they should only be concerned with having a boolean property "IsSpellPrepared" set to true;